cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java

```diff
@@ -663,6 +711,14 @@ private boolean processAddNodeLocally(Node node, AddNodeResponse response,
     return false;
   }
 
+
+  private void sendStatusToNewNode(Node node, CheckStatusRequest checkStatusRequest,
+      AsyncMethodCallback<CheckStatusResponse> response)
+      throws TException {
+    AsyncClient client = (AsyncClient) connectNode(node);
+    client.checkStatus(checkStatusRequest, response);
```

Comment:
client.checkStatus is an asynchronous method, so if you use it like this, you will probably get nothing in the response. Please refer to other usages where a lock (`synchronized`) is used to assure that the result is returned before we move on.

Reply:
Thanks for your suggestion. I will implement it.